specs/rails/run_rspec_tests.yaml

```diff
@@ -0,0 +1,15 @@
+---
+name: Runs rspec tests 
+command: |-
+    bundle exec rspec {{test_path}}
+tags:
+  - rails
+description: Uses bundler to run all the test suite or the test from the path used as parameter
```

Comment:
I'm thinking:
"Uses bundler to run the entire test suite or a set of tests (from the test_path parameter)"